package.json

```diff
@@ -18,6 +18,10 @@
 	},
 	"dependencies": {
 		"@prisma/client": "^6.11.1",
+		"@radix-ui/react-dialog": "^1.1.14",
+		"@radix-ui/react-separator": "^1.1.7",
+		"@radix-ui/react-slot": "^1.2.3",
+		"@radix-ui/react-tooltip": "^1.2.7",
```

Comment:
Interesting choice of UI lib. Heard about this but never used. Any particular reason for choosing it? Have you used it before?